src/app/application.cpp

```diff
@@ -437,8 +437,8 @@ void Application::runExternalProgram(const BitTorrent::Torrent *torrent) const
 
     // The processing sequenece is different for Windows and other OS, this is intentional
 #if defined(Q_OS_WIN)
-    const QString program = replaceVariables(Preferences::instance()->getAutoRunProgram().trimmed());
-    const std::wstring programWStr = program.toStdWString();
+    const QString program = replaceVariables(programTemplate);
+    const std::wstring programWStr = replaceVariables(program).toStdWString();
```

Comment:
This line seems applied incorrectly, it should be:
```suggestion
    const std::wstring programWStr = program.toStdWString();
```

Reply:
Fixed.